niche/generate-docs.bashlib

```diff
@@ -76,7 +76,7 @@ generate-docs_create_tarball() {
 	# within the tarball for posterity.
 	cp {${0},$(dirname ${BASH_SOURCE}/generate-docs.bashlib)} "${dir}"/${P}-docs/ || { eerror "Failed to embed script within tarball. Exiting!" ; exit 1 ; }
 
-	tar -caf "${dir}"/${P}-docs.tar.xz -C "${dir}" ${P}-docs/
+	tar --owner=0 --group=0 -caf "${dir}"/${P}-docs.tar.xz -C "${dir}" ${P}-docs/
```

Comment:
Maybe copy the other options from https://github.com/thesamesam/sam-gentoo-scripts/blob/main/maint/bump-go#L26?